start_ocean.sh

```diff
@@ -43,14 +54,22 @@ while :; do
             printf $COLOR_Y'Starting and reusing the database ...\n\n'$COLOR_RESET
             ;;
         --no-pleuston)
-            COMPOSE_FILE='docker-compose-no-pleuston.yml'
+            COMPOSE_FILE="$DIR/docker-compose-no-pleuston.yml"
             printf $COLOR_Y'Starting without Pleuston...\n\n'$COLOR_RESET
             ;;
         --local-parity-node)
             export KEEPER_NETWORK_NAME="ocean_poa_net_local"
-            COMPOSE_FILE='docker-compose-local-parity-node.yml'
+            COMPOSE_FILE="$DIR/docker-compose-local-parity-node.yml"
             printf $COLOR_Y'Starting with local Parity node...\n\n'$COLOR_RESET
             ;;
+        --kovan-parity-node)
+            [ -f "$KOVAN_ADDRESS_FILE" ] || error "Kovan account json file not found in $KOVAN_ADDRESS_FILE"
+            [ -f "$KOVAN_PASSWORD_FILE" ] || error "Kovan account password file not found in $KOVAN_PASSWORD_FILE"
+            [ -z "$UNLOCK_ADDRESS" ] && error "Kovan account address must be exported in variable \$UNLOCK_ADDRESS"
+            COMPOSE_FILE='docker-compose-only-parity.yml'
```

Comment:
```suggestion
            COMPOSE_FILE="$DIR/docker-compose-only-parity.yml"
```

Reply:
Also, there doesn't seem to be a file named `docker-compose-only-parity.yml`

Reply:
Also added the forgotten file! Thanks for pointing!